java/conf/cobbler/snippets/sles_register

```diff
@@ -8,13 +8,7 @@ if [ -x /usr/bin/curl ]; then
 else
     wget http://$redhat_management_server/pub/RHN-ORG-TRUSTED-SSL-CERT -O $mycert
 fi
-if [ -x /usr/sbin/update-ca-certificates -a -d /etc/pki/trust/anchors ]; then
-    ln -s $mycert /etc/pki/trust/anchors/RHN-ORG-TRUSTED-SSL-CERT.pem
-    /usr/sbin/update-ca-certificates
-else
-    ln -s $mycert /etc/ssl/certs/RHN-ORG-TRUSTED-SSL-CERT.pem
-    c_rehash
-fi
+/usr/share/rhn/certs/update-ca-cert-trust.sh $mycert
```

Comment:
This will not work as spacewalk-certs-tools is not installed on clients.
It is a package for Server and Proxy.
We better keep the original code here.